src/olympia/addons/serializers.py

```diff
@@ -276,8 +298,8 @@ def fake_object(self, data):
         self._attach_fields(
             obj, data,
             ('average_daily_users', 'bayesian_rating', 'created',
-             'default_locale', 'guid', 'hotness', 'icon_type',
-             'is_experimental', 'is_listed',
+             'default_locale', 'guid', 'has_eula', 'has_privacy_policy',
```

Comment:
This too looks like it would be better as a single-item-per-line list.

Unrelated but "hotness" is such a weird attributes name, I'd love to rename it.